build_image.sh

```diff
@@ -31,7 +31,7 @@ cd $app_path/workspace/docker_certificates
 cd $app_path
 
 echo "-- Running docker build command"
-docker build -t $app_name:${setup_type} -f $parent_path/docker/ccf_app_js.${setup_type} .
+docker build -t $app_name:${setup_type} -f $app_path/docker/ccf_app_js.${setup_type} .
 
 echo "-- Removing host config files copies"
 rm $app_path/workspace/docker_certificates/cchost_config_${setup_type}_js.json
```

Comment:
You don't need this rm if we get rid of the cp

Reply:
Acknowledged!

Reply:
Done.